extra/plzip/build

```diff
@@ -5,6 +5,10 @@
 # compile plzip.
 (cd lzlib; ./configure; make)
 
+case "$(${CC:-cc} -print-file=libc++.a)" in
```

Comment:
This should be `-print-file-name=`?

Reply:
Aye. Should be `-print-file-name`. Oops.